src/ducks/sharing/index.js

```diff
@@ -38,6 +38,44 @@ export const createPermSet = (id, doctype) =>
     }
   })
 
+export const share = async (id, doctype, email, url) => {
+  const target = await cozy.client.fetchJSON('POST', '/sharings/recipient', {
+    email,
+    url
+  })
+
+  const sharings = await cozy.client.fetchJSON('POST', '/sharings/', {
+    desc: 'Share Ilyn Sarkisian',
+    permissions: {
+      album: {
+        description: 'album',
+        type: doctype,
+        values: [
+          id
+        ]
```

Comment:
I did not rely on the documentation which need to be updated as @Gara64 and julien said. So I cargo culted the https://github.com/Gara64/Sharotronic code.